File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_ViewCustomListRowFieldRow.cpp

#include "LowLevelFields_ViewCustomListRowFieldRow.h"
#include "LowLevelFields_ViewCustomListRowFieldRow.h"
#include <vector>
#include <string>

namespace widgetassertions
{
  std::string LowLevelFields_ViewCustomListRowFieldRow::getHeaderLabelText( ) 
  {
    return this->HeaderLabelText;
  }
  std::vector<std::string> LowLevelFields_ViewCustomListRowFieldRow::getAdditionalStrings( ) 
  {
    return this->AdditionalStrings;
  }
  std::string LowLevelFields_ViewCustomListRowFieldRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_ViewCustomListRowFieldRow.cs
/// <filename>
///     LowLevelFields_ViewCustomListRowFieldRow.cs
/// </filename>
using System.Collections.Generic;

namespace widgetassertions
{
    public class LowLevelFields_ViewCustomListRowFieldRow
  {
        private string HeaderLabelText;
     
        public     virtual string getHeaderLabelText()
    {
      return this.HeaderLabelText;
    }
     
        public System.Collections.Generic.List<string> AdditionalStrings = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getAdditionalStrings()
    {
      return this.AdditionalStrings;
    }
     
        private string RowHandle;
     
        public     virtual string getRowHandle()
    {
      return this.RowHandle;
    }
  }
}

